gcc/rust/rust-session-manager.cc

```diff
@@ -346,7 +401,13 @@ Session::handle_option (
     case OPT_frust_crate_:
       // set the crate name
       if (arg != nullptr)
-	ret = options.set_crate_name (arg);
+	{
+	  Error error ((Location ()), std::string ());
```

Comment:
```suggestion
	  Error error (Location (), "");
```

Reply:
I tried this, and CI/bors did not like that. See https://github.com/Rust-GCC/gccrs/runs/5883397729?check_suite_focus=true#step:6:17.

Reply:
Oh okay, fair! I also suggested removing the extra parentheses around the `Location ()` constructor. By the way, if you're having trouble building the project or run the test suite on your machine let us know on Zulip :) We'll be happy to help

Reply:
> Oh okay, fair! I also suggested removing the extra parentheses around the `Location ()` constructor. By the way, if you're having trouble building the project or run the test suite on your machine let us know on Zulip :) We'll be happy to help

I can run the test suites on my local machine. However, I discovered that sometimes CI is pickier about this, probably due to a newer GCC version.

Reply:
Hmm, my GCC 11.1 on my system does not like removing the parenthesis as well:

```
../../gcc/rust/rust-session-manager.cc:406:23: warning: parentheses were disambiguated as a function declaration [-Wvexing-parse]
  406 |           Error error (Location (), std::string ());
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Oh that's interesting! First time I see the [most vexing parse](https://en.wikipedia.org/wiki/Most_vexing_parse) in the wild :D You can either keep the parentheses or turn that line into
```suggestion
	  auto error = Error (Location (), std::string ());
```
Whichever you prefer. I like the `auto` keyword personally :D